packages/node/src/Session.spec.ts

```diff
@@ -167,16 +154,11 @@ describe("Session", () => {
       const mySession = new Session({
         clientAuthentication: mockClientAuthentication(),
       });
-      // Spy on the private session logout
-      const spiedLogout = jest.spyOn(
-        mySession as unknown as { internalLogout: () => Promise<void> },
-        "internalLogout",
-      );
       const logoutEventcallback = jest.fn();
       mySession.events.on(EVENTS.LOGOUT, logoutEventcallback);
       (mySession.events as EventEmitter).emit(EVENTS.SESSION_EXPIRED);
       // The internal logout should have been called...
-      expect(spiedLogout).toHaveBeenCalled();
+      expect(mySession.info.isLoggedIn).toBe(false);
```

Comment:
It is intentional that we no longer log out (which clears the storage), but rather just update the logged in status.